packages/core/src/codewhispererChat/controllers/chat/tabBarController.ts

```diff
@@ -88,7 +88,9 @@ export class TabBarController {
                 selectedTab.historyId,
                 selectedTab.tabType,
                 selectedTab.conversations.flatMap((conv: Conversation) =>
-                    conv.messages.map((message) => messageToChatItem(message))
+                    conv.messages
+                        .filter((message) => message.shouldDisplayMessage !== false)
```

Comment:
nit: just `message.shouldDisplayMessage` should be enough?

Reply:
Since older history messages didn't include this flag, we won't be able to load those older tabs